OracleManagementAgent/dockerfiles/latest/Dockerfile

```diff
@@ -25,13 +25,16 @@
 #		$ bash build.sh
 #
 
-FROM oraclelinux:7-slim
+FROM container-registry.oracle.com/os/oraclelinux:8-slim
```

Comment:
```suggestion
FROM oraclelinux:8
```

Reply:
We want to make sure it is pulled from Oracle container registry and it has to be the Oracle 8 slim image which is what we are approved for use.

Reply:
Customers have no such restrictions and as such, we should be using the official Oracle Linux images from Docker Hub, which is what customers generally expect. Note as well that the Oracle Linux images on Docker Hub, GitHub Container Registry and Oracle Container Registry are identical, as they are all built from the same rootfs tarballs published by Oracle at https://github.com/oracle/container-images